draft-terbu-sd-jwt-vc.md

```diff
@@ -62,72 +52,70 @@ the Subject, for example to prove eligibility to access certain services.
            |            |
            +------------+
                  |
-          Issues SD-JWT VC
-     and Issuer-Issued Disclosures
+      Issues Verifiable Credential
                  |
                  v
-           +------------+                                 +------------+
-           |            |                                 |   Status   |
-           |   Holder   |-------- optionally ------------>|  Provider  |
-           |            |       retrieves status          |            |
-           +------------+             VC                  +------------+
+           +------------+
+           |            |
+           |   Holder   |
+           |            |
+           +------------+
                  |
-         Presents SD-JWT VP
-   and Holder-Selected Disclosures
+    Presents Verifiable Credential
                  |
                  v
            +-------------+
-           |             |+
-           |  Verifiers  ||+
-           |             |||
-           +-------------+||
-            +-------------+|
+           |             |+                                  +------------+
+           |  Verifiers  ||+                                 |   Status   |
+           |             |||---------- optionally ---------->|  Provider  |
+           +-------------+||       retrieve credential       |            |
+            +-------------+|            status               +------------+
              +-------------+
 ~~~
-Figure: SD-JWT VC Issuance and SD-JWT VP Presentation Flow w/ Status
-
-In the three-party-model, Verifiers have to trust Issuers to make
-trustworthy statements about the Subject and they can additionally require that
-the Holder provides a proof that they are the intended Holder of the Verifiable
-Credential which can important for security reasons. This is only possible if
-an Issuer binds the Verifiable Credential to a specific Holder at the time of
-issuance. This process is referred to as Holder Binding and is further
+Figure: Three-Party-Model with optional Status Provider
+
+Verifiers can check the authenticity of the data in the Verifiable Credentials
+and optionally enforce Holder Binding, i.e., ask the Holder to prove that they
+are the intended holder of the Verifiable Credential by proving possession of a
+cryptographic key referenced in the credential. This process is further
 described in [@!I-D.ietf-oauth-selective-disclosure-jwt].
 
+To support revocation of credentials, an optional fourth party can be involved,
+a Status Provider, who delivers revocation information to Verifiers.
```

Comment:
```suggestion
a Status Provider, who delivers credential status information to Verifiers.
```

Reply:
We need to say what "credential status" is. Otherwise the term remains very unclear.

Reply:
I thought that this would be covered by @paulbastian spec?

Reply:
I'm not talking about *how* it is achieved, but what "status" means.

Reply:
Alright